src/components/movies/MovieList.vue

```diff
@@ -19,7 +20,54 @@ export default {
   components: {
     MovieCard,
   },
+  data() {
+    return {
+      selectedGenre: "",
+    };
+  },
+  props: {
+    search: {
+      type: String,
+      default: "",
+    },
+  },
+  computed: {
+    searchedMovies() {
+      let re = new RegExp(this.search, "i");
+      return this.$store.getters.movieList.filter((movie) =>
+        movie.title.match(re)
+      );
+    },
+    moviesByCategory() {
+      const moviesByCategory = this.searchedMovies.filter(
+        (movie) => movie.genre.name === this.$store.getters.selectedGenre
+      );
+      return this.$store.getters.selectedGenre == "All Movies" ||
+        this.$store.getters.selectedGenre == ""
+        ? this.searchedMovies
+        : moviesByCategory;
+    },
+  },
 };
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+.movie-list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
You can simplify it like this using the `repeat` 🧙:
```suggestion
  grid-template-columns: repeat(3, 1fr);
```